Client/Frontend/Settings/BraveCoreDebugSwitchesView.swift

```diff
@@ -30,6 +30,9 @@ extension BraveCoreSwitchKey {
     case .p3aUploadServerURL:
       return "Upload Server URL"
     default:
+      if rawValue == "enable-features" {
```

Comment:
Can you add a definition for this somewhere and use that instead of handling raw value like this? Like 

```swift
extension BraveCoreSwitchKey {
  static let enableFeatures: BraveCoreSwitchKey = .init(rawValue: "enable-features")
}
```

Reply:
Fixed in [d89d046](https://github.com/brave/brave-ios/pull/6728/commits/d89d04697bd1f6ae5f7e70e1c398ab84e540db3f)